src/main/java/com/avp/homework4_conditionOperators/task5_LoginService/service/UserLoginService.java

```diff
@@ -0,0 +1,49 @@
+package com.avp.homework4_conditionOperators.task5_LoginService.service;
+
+import com.avp.homework4_conditionOperators.task5_LoginService.bean.User;
+
+public class UserLoginService {
+
+    public boolean login(User user, String password) {
+        if (comparePasswords(user, password)) {
+            if (!isUserBlocked(user)) {
+                resetCountOfLoginAttempts(user);
+                return true;
+            }
+            return false;
+        } else {
+            decrementCountOfLoginAttempts(user);
+            if(!IsLoginAttemptsLeft(user)) {
+                blockUser(user);
+            }
+            return false;
+        }
+    }
+
+    public boolean comparePasswords(User user, String password) {
+        return user.getPassword().equals(password);
+    }
+
+    public User blockUser(User user) {
+        user.setBlocked(true);
+        return user;
+    }
+
+    public User resetCountOfLoginAttempts(User user){
+        user.setCountOfLoginAttempts(3);
+        return user;
+    }
+
+    public boolean isUserBlocked(User user) {
+        return user.isBlocked();
+    }
+
+    public User decrementCountOfLoginAttempts(User user) {
+        user.setCountOfLoginAttempts(user.getCountOfLoginAttempts() - 1);
+        return user;
+    }
+
+    public boolean IsLoginAttemptsLeft(User user) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Done